aether-common-module/aether/common/conf/settings.py

```diff
@@ -245,27 +245,22 @@
 # Storage Configuration
 # ------------------------------------------------------------------------------
 
-CUSTOM_STORAGE = os.environ.get('DJANGO_REMOTE_STORAGE', '')
-
-if CUSTOM_STORAGE:   # pragma: no cover
-    INSTALLED_APPS += ['storages', ]
-    STATIC_FILES = bool(os.environ.get('REMOTE_STATIC_FILES'))
-    if CUSTOM_STORAGE.lower() == 's3':
-        DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
-        AWS_AUTO_CREATE_BUCKET = True
-        AWS_STORAGE_BUCKET_NAME = os.environ['BUCKET_NAME']
-        if STATIC_FILES:
-            STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
-    elif CUSTOM_STORAGE.lower() == 'gcs':
-        DEFAULT_FILE_STORAGE = 'storages.backends.gs.GSBotoStorage'
-        GS_AUTO_CREATE_BUCKET = True
-        GS_BUCKET_NAME = os.environ['BUCKET_NAME']
-        if STATIC_FILES:
-            STATICFILES_STORAGE = 'storages.backends.gs.GSBotoStorage'
-
-    logger.info('{} file system used!'.format(CUSTOM_STORAGE))
+DJANGO_STORAGE_BACKEND = os.environ['DJANGO_STORAGE_BACKEND']
+
+if DJANGO_STORAGE_BACKEND == 'filesystem':
+    DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+elif DJANGO_STORAGE_BACKEND == 's3':
+    INSTALLED_APPS += ['storages']
```

Comment:
Minor: I would add it to the `INSTALLED_APPS` list and just keep it there. The overhead is minimum, and it will only be for development.

Reply:
Agreed, PR updated.